runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx

```diff
@@ -118,9 +117,12 @@ const ActionList: FC<ActionListProps> = ({
                                         item={item}
                                         disabled={
                                             !hasAdvancedActionsAccess &&
-                                            ADVANCED_ACTION_IDS.includes(
+                                            (ADVANCED_ACTION_IDS.includes(
                                                 item.id
-                                            )
+                                            ) ||
+                                                ADVANCED_ACTION_GROUP_IDS.includes(
+                                                    key
+                                                ))
```

Comment:
(ADVANCED_ACTION_IDS.includes(item.id) 
|| ADVANCED_ACTION_GROUP_IDS.includes(key))

Reply:
To akurat sprawka prettier XD

Reply:
ha! miecz obusieczny